dymos/transcriptions/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .analytic.analytic import Analytic
 from .explicit_shooting import ExplicitShooting
 from .pseudospectral.gauss_lobatto import GaussLobatto
-from .pseudospectral.radau_pseudospectral import Radau
+from .pseudospectral.radau_pseudospectral import Radau as RadauDeprecated
```

Comment:
I wonder if it would be better to rename the module to radau_deprecated, just to alleviate future when viewing the file in the future.  Then radau_new can become radau_pseudospectral.  Otherwise, kind of mixing paradigms here.